pypesto/petab/importer.py

```diff
@@ -417,6 +424,31 @@ def create_objective(
             # FIXME: currently not supported with hierarchical
             kwargs['guess_steadystate'] = False
 
+        if self._ordinal:
+            # TODO add constants to C
```

Comment:
todo

Reply:
This is also in part a question: should these strings be in the constants file, as they're not something used throughout, but just a name of variable passed for constructors? (like 'guess_steadystate' is)

Reply:
so this is about e.g. `inner_problem_method`? if it's just an argument that is used in only exactly this function, fine to keep it as a string literal. only as soon as you use the string literal >1 times, make it a variable.

Reply:
+ document this argument in the method docstring

Reply:
Ok, won't add to C then and will add to kwargs part of docstring.